src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2446,7 +2446,7 @@ namespace HVACUnitarySystem {
 			PrintFlag = true;
 			TempSize = UnitarySystem( UnitarySysNum ).DesignMaxOutletTemp;
 			FieldNum = 18; // N18, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
-			SizingString = UnitarySystemNumericFields( UnitarySysNum ).FieldNames( FieldNum ) + " [m3/s]";
+			SizingString = UnitarySystemNumericFields( UnitarySysNum ).FieldNames( FieldNum ) + " [C]";
```

Comment:
should also be either autosizable or default but not both. No autosize if there is a decent default in my opinion.  

So yest it does seems like it should be N17 here, and reporting of MaxOATSuppHeat for N18 is not happening anywhere.